src/metabase/integrations/ldap.clj

```diff
@@ -177,16 +177,21 @@
      :filter     (str/replace (ldap-user-filter) filter-placeholder (Filter/encodeValue username))
      :size-limit 1})))
 
+(defn- lower-case-attribute-names
+  "Changes the attribute names of the given LDAP record to lower case."
+  [record]
+  (into {} (for [[key value] record] [(keyword (u/lower-case-en (name key))) value])))
```

Comment:
Just some minor tweaks to improve readability:

```clojure
(defn- lower-case-attribute-names
  "Changes the attribute names of the given LDAP record to lower case."
  [record]
  (into {} (for [[k v] record]
             [(-> k name u/lower-case-en keyword) v])))
```

1. Adding a newline after the `for`'s bindings is expected, and makes it a lot easier read.
2. The treading macro `->` can save you a few parentheses. It turns `(h (g (f x)))` into `(-> x f g h)`.
3. Using `k` and `v` instead of `key` and `value` avoids shadowing the built-in `key` function. Such variable names are understood as [idiomatic](https://guide.clojure.style/#idiomatic-names) in Clojure.

Reply:
I think I applied all of the above.